### PR TITLE
Add a unit test which validates parsing primitive types.

### PR DESCRIPTION
**Pull request by dinowernli**

### CODE REVIEW

File: src/test/java/me/dinowernli/grpc/polyglot/io/MessageReaderTest.java [hunk 1]
```diff
@@ -56,6 +54,13 @@ public void handlesEmptyStream() {
   }
 
   @Test
+  public void handlePrimitives() {
+    reader = MessageReader.forFile(dataFilePath("request_with_primitives.pb.ascii"), DESCRIPTOR);
+    ImmutableList<DynamicMessage> result = reader.read();
+    assertThat(result).isNotEmpty();
```
Comment (RaganH) [inline src/test/java/me/dinowernli/grpc/polyglot/io/MessageReaderTest.java]: Slightly odd that you don't assert that the number field has the correct value here.

**Comment by RaganH** [comment]
LGTM, though as I said via e-mail I don't think this was the cause of the issue I was seeing.

File: src/test/java/me/dinowernli/grpc/polyglot/io/MessageReaderTest.java (hunk 1 above)
Comment (dinowernli) [reply src/test/java/me/dinowernli/grpc/polyglot/io/MessageReaderTest.java]: Agreed. Fixed.

**Comment by dinowernli** [comment]
thanks @RaganH

merging once it goes green, seems like an ok test to have regardless

**Pull request merged**